doc/default.nix

```diff
@@ -3,6 +3,10 @@ let
   lib = pkgs.lib;
   sources = lib.sourceFilesBySuffices ./. [".xml"];
   sources-langs = ./languages-frameworks;
+  libDocJSON = builtins.toFile "options.json" (builtins.toJSON (import ./fn-docs.nix));
+  libDocBook= pkgs.runCommand "options-db.xml" {} ''
+    ${pkgs.ruby}/bin/ruby ${./lib2docbook.rb} < ${libDocJSON} > $out
+  '';
```

Comment:
All fixed in https://github.com/NixOS/nixpkgs/pull/23505/commits/a801c0bd1a796a21c0399d823f8a6bfb9d63009e. Thanks!